app/javascript/mastodon/containers/account_relationship_button_container.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { defineMessages, injectIntl, FormattedMessage } from 'react-intl';
+import AccountRelationshipButton from '../components/account_relationship_button';
+import {
+  followAccount,
+  unfollowAccount,
+  blockAccount,
+  unblockAccount,
+  muteAccount,
+  unmuteAccount,
+} from '../actions/accounts';
+import { openModal } from '../actions/modal';
+import { initMuteModal } from '../actions/mutes';
+import { unfollowModal } from '../initial_state';
+import { makeGetAccount } from '../selectors';
+
+const messages = defineMessages({
+  unfollowConfirm: { id: 'confirmations.unfollow.confirm', defaultMessage: 'Unfollow' },
+});
+
+const makeMapStateToProps = () => {
+  const getAccount = makeGetAccount();
+
+  const mapStateToProps = (state, props) => ({
```

Comment:
The second argument is named `props` in `mapStateToProps`, but it is destructed in `mapDispatchToProps`. It is better to keep them consistent.

Reply:
Sorry, what should I do specifically?

Reply:
Replace `props` with `{ id }`. You will be able to refer to the ID just with `id` instead of `props.id`. That's what `mapDispatchToProps` does.